### PR TITLE
update provider-azure owners

### PR DESCRIPTION
**Pull request by andyzhangx**
remove @craiglpeters and add myself to provider-azure owners

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -170,7 +170,7 @@ aliases:
     - nckturner
     - wongma7
   provider-azure:
-    - craiglpeters
+    - andyzhangx
     - justaugustus
```
Comment (mrbobbytables) [inline OWNERS_ALIASES:174]: Can be handled in a separate PR, but @justaugustus do you want to remain as an approver for azure?

Comment (justaugustus) [reply OWNERS_ALIASES:174]: @mrbobbytables -- Feel free to remove me! This is vestigial from SIG Azure days.

Comment (andyzhangx) [reply OWNERS_ALIASES]: fixed, thanks folks for the confirm.

**Review by mrbobbytables — COMMENTED** [review]
/label tide/merge-method-squash
/approve

**Pull request merged**